### DISCUSSION TITLE
Cannot create db branch

### DISCUSSION THREAD

**Discussion by marwan-at-work** [planetscale/discussion#50] [Bug Reports] [answered]
Due to https://github.com/planetscale/beta/discussions/48, I thought I'd create a new database to unblock myself. However, I cannot create my first branch to create a schema change: 

<img width="1792" alt="Screen Shot 2021-05-26 at 10 09 09 PM" src="https://user-images.githubusercontent.com/16294261/119755211-a0999880-be6f-11eb-8725-5ec8abe140ac.png">

**Comment by Anders-PlanetScale** [comment, marked as answer]
@marwan-at-work,
I think the block should be removed from your account as part of the ongoing work in https://github.com/planetscale/beta/discussions/48
If that is not the case please let us know.
